support-frontend/assets/components/checkoutBenefits/checkoutBenefitsListData.tsx

```diff
@@ -1,13 +1,13 @@
 import type { SerializedStyles } from '@emotion/react';
 import { css } from '@emotion/react';
-import { neutral } from '@guardian/source-foundations';
+import { palette } from '@guardian/source-foundations';
```

Comment:
Just a general question, do we prefer to use the palette going forward?

Reply:
Yes, the direct use of `neutral`, `brand` etc is deprecated in the most recent version of Source so we should be using `palette` (this should get flagged in VSCode so it's hard to miss).